esp32-wroom-rp/src/gpio.rs

```diff
@@ -80,32 +77,35 @@ where
 {
     fn init(&mut self) {
         // Chip select is active-low, so we'll initialize it to a driven-high state
-        self.cs.set_high().unwrap();
+        self.cs.set_high().ok().unwrap();
+        self.gpio0.set_high().ok().unwrap();
+        self.resetn.set_high().ok().unwrap();
+        self.get_esp_ready();
     }
 
-    fn reset<D: DelayUs>(&mut self, delay: &mut D) {
-        self.gpio0.set_high().unwrap();
-        self.cs.set_high().unwrap();
-        self.resetn.set_low().unwrap();
-        delay.delay_ms(10).ok().unwrap();
-        self.resetn.set_high().unwrap();
-        delay.delay_ms(750).ok().unwrap();
+    fn reset<D: DelayMs<u16>>(&mut self, delay: &mut D) {
+        self.gpio0.set_high().ok().unwrap();
+        self.cs.set_high().ok().unwrap();
+        self.resetn.set_low().ok().unwrap();
```

Comment:
Curious about the addition of the `ok()` functions here.

Reply:
It's because I went back to the older version of embedded-hal. It'll complain about an unused Result.

Reply:
Got it. Thank you for explaining.